lib/tasks/test_vmdb.rake

```diff
@@ -2,15 +2,99 @@ require_relative "./evm_test_helper"
 
 if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
 namespace :test do
-  namespace :vmdb do
+  specs = {
+    automation: 149,   # automation
+    controllers: 1274, # controllers
+    helpers: 1005,     # ui
+    initializers: 1,   # others
+    lib: 1283,         # lib
+    mailers: 19,       # lib
+    migrations: 321,   # migrations
+    models: 4494,      # models
+    presenters: 122,   # ui
+    replication: 2,    # replication
+    requests: 797,     # others
+    routing: 1769,     # routing
+    services: 122,     # others
+    task_helpers: 4,   # others
+    tools: 46,         # others
+    views: 65,         # ui
+  }
```

Comment:
yea, that was just a note for me to see how the split would make sense.
In case we go that route I'll DRY things up in here :smile: